bin/setup

```diff
@@ -14,14 +14,14 @@ Dir.chdir APP_ROOT do
   # Add necessary setup steps to this file:
 
   puts %q[
-   _             _
-  | |           (_)
-  | | ___   __ _ _ _ __    __ _  _____   __
-  | |/ _ \ / _` | | '_ \  / _` |/ _ \ \ / /
-  | | (_) | (_| | | | | || (_| | (_) \ V /
-  |_|\___/ \__, |_|_| |_(_)__, |\___/ \_/
-            __/ |          __/ |
-           |___/          |___/
+    
+    ______               _____
+    ___  / _____________ ___(_)______   _______ _________   __
+    __  /  _  __ \_  __ `/_  /__  __ \  __  __ `/  __ \_ | / /
+    _  /___/ /_/ /  /_/ /_  / _  / / /___  /_/ // /_/ /_ |/ /
+    /_____/\____/_\__, / /_/  /_/ /_/_(_)\__, / \____/_____/
+                 /____/                 /____/
+          |___/
```

Comment:
💯

Reply:
now with 20% more 🆒

Reply:
I like how many comments this is getting.

Reply:
🥇